src/Tizen.NUI/src/public/BaseComponents/Style/ViewStyle.cs

```diff
@@ -26,7 +26,7 @@ namespace Tizen.NUI.BaseComponents
     /// </summary>
     /// This will be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API.
     [EditorBrowsable(EditorBrowsableState.Never)]
-    public partial class ViewStyle : BindableObject
+    public partial class ViewStyle : BindableObject, IDisposable
```

Comment:
The `ControlStyle` class inherits `ViewStyle` and it implements IDisposable already.
Please clean up IDisposable code from the `ControlStyle`.

Reply:
The parent inheritance has been changed to implement Disposable.